bindings/pyroot_experimental/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${py_localruntimedir})
+set(CMAKE_INSTALL_LIBDIR ${CMAKE_INSTALL_PYROOTDIR})
```

Comment:
we set this for the root standard library package macro to install libraries in the right directory